pkg/webservice/handlers_test.go

```diff
@@ -1781,6 +1784,15 @@ func TestGetApplicationHandler(t *testing.T) {
 	getApplication(resp4, req4)
 	err = json.Unmarshal(resp4.outputBytes, &appsDao4)
 	assert.NilError(t, err, unmarshalError)
+	assert.Assert(t, appsDao4 != nil)
+	assert.Equal(t, appsDao4.ApplicationID, "app-1")
+	assert.Equal(t, appsDao4.Partition, "default")
+	assert.Equal(t, appsDao4.QueueName, "root.default")
+	if !appsDao4.HasReserved {
+		assert.Equal(t, len(appsDao4.Reservations), 0)
+	} else {
+		assert.Check(t, len(appsDao4.Reservations) > 0, "app should have at least 1 reservation")
+	}
```

Comment:
What determines which branch is taken? This is not a re-used code as far as I can see, so this is supposed to be predictable.

Reply:
My question also applies to the existing code at L#1748.

Reply:
Indeed, and looks like there's no reservation there, so I assert it to be the default value.
```
assert.Equal(t, len(appsDao4.Reservations), 0)
```

Reply:
Done.